runtime/src/lib.rs

```diff
@@ -269,10 +269,12 @@ impl pallet_sudo::Trait for Runtime {
 // Configure pallet constants
 parameter_types! {
     pub const AdminRole: u8 = ADMIN_ROLE;
+    pub const PilotRole: u8 = PILOT_ROLE;
```

Comment:
PiliotRole is unnecessary here. This parameter is not used for configuration the pallet.